Docs/neurons.md

```diff
@@ -161,5 +161,20 @@ neurons:
 
 Here, the first neuron will use the default tts as set in the settings.yml file. The second neuron will use the tts "acapela".
```

Comment:
Yes, it's part of original doc:


```
You can override the default tts for each neurons. Just add the "tts" parameter to the neuron like in the example bellow

neurons:
  - say:
    message:
      - "Hello, sir"
  - say:
    message:
      - "My name is Kalliope"
    tts: "acapela"

Here, the first neuron will use the default tts as set in the settings.yml file. The second neuron will use the tts "acapela".
```

if you want i move this sentence on top of exemple